sdk/easm/azure-defender-easm/samples/README.md

```diff
@@ -0,0 +1,3 @@
+# Azure EASM samples for Python client
+
+The sample programs here demonstrate some common use case scenarios for the Azure EASM Python SDK Client.
```

Comment:
Not blocking for this PR - but it would be good to add an index table to the readme that summaries which each sample demonstrates.

Reply:
I've added a table of contents to the readme